src/Servant/Extra/ContentTypes.hs

```diff
@@ -0,0 +1,83 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE KindSignatures #-}
+{-# LANGUAGE MultiParamTypeClasses #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TupleSections #-}
+
+-- | Extra content types for Servant.
+--
+module Servant.Extra.ContentTypes
+    ( CBOR
+    , Packed
+    , WithHash
+    ) where
```

Comment:
I'd rather have one module for now as `Servant.ContentTypes.CBOR`, exposing the content of both this one and the `Codec.CBOR.Extra`. We don't use them separately for now so let's not introduce needless indirection at this stage.

Reply:
I've now merged these two modules, as suggestion.